bin/run.sh

```diff
@@ -33,28 +33,23 @@ echo "${slug}: testing..."
 
 # Run the tests for the provided implementation file and redirect stdout and
 # stderr to capture it
-test_output=$(false)
-# TODO: substitute "false" with the actual command to run the test:
-# test_output=$(command_to_run_tests 2>&1)
+cd "${solution_dir}"
+echo "${solution_dir}"
+test_output=$(sqlite3 -bail < "./${slug}_test.sql" 2>&1)
 
 # Write the results.json file based on the exit code of the command that was 
 # just executed that tested the implementation file
-if [ $? -eq 0 ]; then
-    jq -n '{version: 1, status: "pass"}' > ${results_file}
+if [ $? -ne 0 ]; then
+    jq -n --arg output "${test_output}" '{version: 3, status: "error", message: $output}' > ${results_file}
 else
-    # OPTIONAL: Sanitize the output
-    # In some cases, the test output might be overly verbose, in which case stripping
-    # the unneeded information can be very helpful to the student
-    # sanitized_test_output=$(printf "${test_output}" | sed -n '/Test results:/,$p')
-
-    # OPTIONAL: Manually add colors to the output to help scanning the output for errors
-    # If the test output does not contain colors to help identify failing (or passing)
-    # tests, it can be helpful to manually add colors to the output
-    # colorized_test_output=$(echo "${test_output}" \
-    #      | GREP_COLOR='01;31' grep --color=always -E -e '^(ERROR:.*|.*failed)$|$' \
-    #      | GREP_COLOR='01;32' grep --color=always -E -e '^.*passed$|$')
-
-    jq -n --arg output "${test_output}" '{version: 1, status: "fail", message: $output}' > ${results_file}
+    if [[ -f "user_output.md" && $(cat "user_output.md" | wc -c) -gt 0 ]]; then
```

Comment:
```suggestion
    if [[ -f "user_output.md" && -s "user_output.md" ]]; then
```
ref https://www.gnu.org/software/bash/manual/bash.html#Bash-Conditional-Expressions

Reply:
This script is sh and not bash. 
```suggestion
    if [ -f "user_output.md" ] && [ ... ]; then
```

Reply:
> This script is sh and not bash.

@IsaacG should we make it bash?

Reply:
I don't see any reason not to make it bash. At which point all `[` should be replaced with `[[` or `((`.